VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -191,4 +194,27 @@ public static void setEndorsedState(@NonNull TextView target,
         }
 
     }
+
+    /**
+     * Renders hyperlinks in the given html to clickable items when applied to the given TextView.
+     *
+     * @param textView The textView which will render the given html.
+     * @param html     The HTML to apply.
+     */
+    public static void linkifyTextView(@NonNull TextView textView, @NonNull String html) {
+        Spanned spannedHtml = (Spanned) DiscussionTextUtils.parseHtml(html);
```

Comment:
`Spanned` is an extension of `CharSequence`. If `parseHtml()` is passing a `Spanned` type to `trim()` then it should get the same type back, and should be able to cast it back to the same (and update the return type signature).